src/tests/FixNoRequireImportsFormatterTests.ts

```diff
@@ -40,7 +40,7 @@ import TestHelper = require('./TestHelper');
 `;
 
         const formatter = new FormatterForTesting(input);
-        formatter.format(TestHelper.runRule(ruleName, null, input).failures);
+        formatter.format(TestHelper.runRule(ruleName, undefined, input).failures);
```

Comment:
These were somehow missed when `no-null-keyword` was turned on (seems like the grunt task isn't linting everything it should), but the type of `userOptions` in the `runRule()` signature changed, so these had to be fixed up anyway.